Makefile

```
@@ -1,18 +1,17 @@
 include ./Makefile.Common
 
 RUN_CONFIG=local/config.yaml
-
 CMD?=
-GIT_SHA=$(shell git rev-parse --short HEAD)
+STATIC_CHECK=staticcheck
+OTEL_VERSION=master
+
 BUILD_INFO_IMPORT_PATH=github.com/open-telemetry/opentelemetry-collector-contrib/internal/version
+GIT_SHA=$(shell git rev-parse --short HEAD)
 BUILD_X1=-X $(BUILD_INFO_IMPORT_PATH).GitHash=$(GIT_SHA)
-ifdef VERSION
+VERSION=$(shell git describe --match "v[0-9]*" HEAD)
```

If you use the following version, you'll allow people to override the version on their make command, making their lives easier:

```suggestion
VERSION ?= $(shell git describe --match "v[0-9]*" HEAD)
```